pkg/virt-handler/ratelimitcmd/pool_test.go

```diff
@@ -0,0 +1,59 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2022 Red Hat, Inc.
+ *
+ */
+
+package ratelimitcmd_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"k8s.io/apimachinery/pkg/types"
+
+	"kubevirt.io/kubevirt/pkg/virt-handler/ratelimitcmd"
+)
+
+var _ = Describe("rate-limited executor pool with exponential-limited-backoff creator", func() {
+
+	const (
+		key1 = types.UID("20007000")
+		key2 = types.UID("20006000")
+	)
+
+	var pool *ratelimitcmd.RateLimitedExecutorPool
+
+	BeforeEach(func() {
+		pool = ratelimitcmd.NewRateLimitedExecutorPool(ratelimitcmd.NewFixedExponentialLimitedBackoffCreator())
+	})
+
+	It("should not override pool element if key1 exists", func() {
+		initialElement := pool.LoadOrStore(key1)
+		Expect(initialElement).To(Equal(pool.LoadOrStore(key1)))
```

Comment:
You can use `BeIdenticalTo` to check identity and not equality.

Reply:
Done